tools/devops/automation/templates/tests/stage.yml

```
@@ -169,3 +176,34 @@ stages:
         statusContext: ${{ parameters.statusContext }}
         vsdropsPrefix: ${{ parameters.vsdropsPrefix }}
         testPrefix: ${{ parameters.testPrefix }}
+
+  - job: publish_skipped_tests
+    ${{ if eq(parameters.parseLabels, true) }}:
+      condition: ne(dependencies.configure.outputs['labels.build-package'], '')
+    ${{ else }}:
+      condition: false
+    displayName: 'GitHub comment - Skipped tests'
+    timeoutInMinutes: 1000
+    dependsOn: # has to wait for the tests to be done AND the data to be uploaded
+    - ${{ if eq(parameters.parseLabels, true) }}:
+      - configure
+
+    pool:
+      vmImage: 'windows-latest'
+      workspace:
+        clean: all
+
+    steps:
+    - pwsh: |
+        Import-Module $Env:SYSTEM_DEFAULTWORKINGDIRECTORY\xamarin-macios\tools\devops\automation\scripts\MaciosCI.psd1
+
+        $gihubComments = New-GitHubCommentsObject -Org "xamarin" -Repo "xamarin-macios" -Token $Env:GITHUB_TOKEN -Hash $Env:GIT_HASH -Debug
+        $sb = [System.Text.StringBuilder]::new()
+        $sb.AppendLine("All tests have been skipped because the label 'skip-all-tests' was set.")
+        $result = $gihubComments.NewCommentFromMessage("Test results", ":bangbang:", $sb.ToString())
```

:bangbang: this is all good, so it seems like we should have a green emojii instead? :seedling:

```suggestion
        $result = $gihubComments.NewCommentFromMessage("Test results", ":seedling:", $sb.ToString())
```